front/src/App.js

```diff
@@ -84,6 +97,17 @@ class App extends Component {
       loggedIn: false
     };
     this.connection = null;
+    this.checkWindowWidth = this.checkWindowWidth.bind(this);
+  }
+
+  checkWindowWidth() {
+    /* 
+     * Once enter the tablet mode, make sure
+     * we transition to the settings page 
+    */
+    if (window.innerWidth >= 768) {
```

Comment:
Works, but is a bit backwards. See comment 2 for my suggestion.

However, if we keep this, could a value from media.js be used instead? This is a magic value.